hapi-utils/src/main/java/com/hederahashgraph/fee/FeeBuilder.java

```diff
@@ -62,6 +62,13 @@
 	public static final int BASIC_ACCOUNT_AMT_SIZE = BASIC_ENTITY_ID_SIZE + LONG_SIZE;
 	public static final int BASIC_TX_ID_SIZE = BASIC_ENTITY_ID_SIZE + LONG_SIZE;
 	public static final int EXCHANGE_RATE_SIZE = 2 * INT_SIZE + LONG_SIZE;
+	public static final int CRYPTO_ALLOWANCE_SIZE = BASIC_ENTITY_ID_SIZE + INT_SIZE + LONG_SIZE; // owner, spender ,
```

Comment:
We are not currently saving the owner Id of the allowance in state right. @tinker-michaelj  Should we still include this?
This would anyway make little to no difference in the price

Reply:
@anighanta  I thought fees is for transaction and it doesn't matter if we are storing the field in state or not.